src/features/emails/components/EmailEditor/EmailSettings/utils/checkBlockErrors.ts

```diff
@@ -0,0 +1,15 @@
+import isURL from 'validator/lib/isURL';
+import { OutputBlockData } from '@editorjs/editorjs';
+
+import { BLOCK_TYPES } from 'features/emails/types';
+
+export default function checkBlockErrors(block: OutputBlockData) {
```

Comment:
Maybe this function could be called `blockHasErrors()` instead, to make it more clear what `true` and `false` means?

Reply:
Great suggestion, thank you!